package.json

```diff
@@ -1,61 +1,61 @@
 {
-  "name": "react-native-maps",
```

Comment:
Why is package.json changed?  if it's format only please remove it from PR, if has anything new please make only that diff to be changed, not the whole file